tests/lora/utils.py

```diff
@@ -2268,7 +2284,7 @@ def test_layerwise_casting_peft_input_autocast_denoiser(self):
             apply_layerwise_casting,
         )
 
-        storage_dtype = torch.float8_e4m3fn
+        storage_dtype = torch.float8_e4m3fn if not torch_device == "mps" else torch.bfloat16
```

Comment:
Same.

Reply:
Done in https://github.com/huggingface/diffusers/pull/10216/commits/e97a83ea7ed53b1ed27834bcbe390bbbc6ab28b3.